nservicebus/messaging/publish-subscribe/index.md

```diff
@@ -168,6 +152,6 @@ NOTE: Version 2.X required a perfect match. This should make it easier to upgrad
 
 In some circumstances it may not be desirable to allow any endpoints to subscribe to a given publisher or event. NServiceBus provides a way to intervene in the subscription process and decide whether a given client should be allowed to subscribe to a given message.
 
-NOTE: Subscription authorization is only available when using transports that require persistence based publish-subscribe.
+NOTE: Subscription authorization is only available when using transports that don't support native publish-subscribe.
```

Comment:
I don't like the negation in the sentence. Shouldn't that be something like

> Subscription authorization is not supported when using transports with native publish-subscribe.

or

> Subscription authorization is only available when using transports with persistence-based message-driven Publish Subscribe.